### PR TITLE
Apply ruff/flake8-implicit-str-concat rule ISC001

### PR DESCRIPTION
**Pull request by DimitriPapadopoulos**
https://docs.astral.sh/ruff/rules/single-line-implicit-string-concatenation/

### CODE REVIEW

**Comment by RonnyPfannschmidt** [comment]
At first glance all occurences where intentionally line split and had gotten cobbled to a single line, i  think multi line Strings are needed

**Comment by DimitriPapadopoulos** [comment]
This is the result of applying black in #255, and `ruff format` will again fold the multiple lines into a single line — unless the code is enclosed in `# fmt: off` and `# fmt: on`.

Would you rather add `# fmt:` directives?

**Comment by RonnyPfannschmidt** [comment]
I think real multi line Strings should be used

**Comment by DimitriPapadopoulos** [comment]
I'll try to do that with `# fmt`.

**Comment by RonnyPfannschmidt** [comment]
what i meant was using tripple quote string, not disabling fmt

File: testing/test_regressions.py [hunk 1]
```diff
@@ -35,7 +35,8 @@ def test_pkginfo_noscmroot(tmp_path: Path, monkeypatch: pytest.MonkeyPatch) -> N
 
     tmp_path.joinpath(".git").mkdir()
     p.joinpath("setup.py").write_text(
-        "from setuptools import setup;" 'setup(use_scm_version={"root": ".."})',
+        '''from setuptools import setup;
```
Comment (DimitriPapadopoulos) [inline testing/test_regressions.py]: Do we want to keep `setup.py` as an one-liner?
```python
from setuptools import setup;setup(use_scm_version={"root": ".."})
```
Or perhaps change as follows?
```python
from setuptools import setup
setup(use_scm_version={"root": ".."})
```

Comment (RonnyPfannschmidt) [reply testing/test_regressions.py]: Good idea

**Comment by RonnyPfannschmidt** [comment]
thanks!

**Pull request merged**